### ISSUE TITLE
Doesn't like flathub in both system and user

### ISSUE THREAD

**Issue by Godsbane** [dragoonDorise/EmuDeck#73]
Hi @dragoonDorise  - ran the upgrade in my (probably) screwy environment. Wanted to share the log.
[emudeck.log](https://github.com/dragoonDorise/EmuDeck/files/8595602/emudeck.log)

```
(deck@steamdeck ~)$ flatpak install flathub net.pcsx2.PCSX2 -y
Looking for matches…
Remote ‘flathub’ found in multiple installations:

   1) system
   2) user

Which do you want to use (0 to abort)? [0-2]: 
```

Pretty sure this became a thing when i installed a beta flatpak from flathub on the command line, but i can't validate because i didn't keep track.

```
(1)(deck@steamdeck ~)$ flatpak remotes --columns=name,title,url,options
Name           Title        URL                                             Options
flathub        Flathub      https://dl.flathub.org/repo/                    system
flathub-beta   Flathub beta https://dl.flathub.org/beta-repo/               system
kdeapps                     http://distribute.kde.org/flatpak-apps-testing/ system
witherking     WitherTech   https://repo.withertech.com/flatpak/            system
flathub        Flathub      https://dl.flathub.org/repo/                    user
sgdboop-origin -            https://dl.flathub.org/build-repo/86647         user,no-enumerate
```

```
(deck@steamdeck ~)$ flatpak install flathub net.pcsx2.PCSX2 -y --system
Looking for matches…
Skipping: net.pcsx2.PCSX2/x86_64/stable is already installed
```

`--system` resolves the query on the command line for me, and matches what the default is for the deck. (validated with a new user.)

We should probably add `--system` explicitly in case any other users have a similar setup or other wonkyness.

**Comment by popsUlfr** [comment]
That's a good point. When you mix and match flatpak installations into the system or user folder, it will start asking where to install packages and indeed `--system` should be explicitly stated.

**Comment by Godsbane** [comment]
#83

**Comment by dragoonDorise** [comment]
This should be fixed on the beta branch

**Comment by dragoonDorise** [comment]
Fixed on V0.17.1!
